templates/patient/insurance/insurance_edit.html.twig

```diff
@@ -0,0 +1,425 @@
+        ,smallform: 'true'
+        ,classNames: "mb-1"
+        ,required: true
+    }, {
+        name: "Subscriber Address"|xl
+        ,data_type: dataTypeText
+        ,field_id: "subscriber_street"
+        ,fld_length: 20
+        ,smallform: 'true'
+        ,classNames: "mb-1 js-capitalize-me"
+        ,required: true
+    }, {
+        name: "Address Line 2"|xl
+        ,data_type: dataTypeText
+        ,field_id: "subscriber_street_line_2"
+        ,fld_length: 20
+        ,smallform: 'true'
+        ,classNames: "mb-1 js-capitalize-me"
+        ,required: false
+    },{
+        name: "City"|xl
+        ,required: true
+        ,data_type: dataTypeText
+        ,field_id: "subscriber_city"
+        ,fld_length: 15
+        ,max_length: 60
+        ,smallform: 'true'
+        ,classNames: "js-capitalize-me mb-1"
+    },{
+        name: "State"|xl
+        ,required: true
+        ,data_type: state_data_type
+        ,field_id: "subscriber_state"
+        ,fld_length: 15
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "js-capitalize-me mb-1 sel2"
+        ,list_id: state_list
+        ,skip: useStateTerminology
+    }, {
+        name: "Locality"|xl
+        ,required: true
+        ,data_type: state_data_type
+        ,field_id: "subscriber_state"
+        ,fld_length: 15
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "js-capitalize-me mb-1 sel2"
+        ,list_id: state_list
+        ,skip: not useStateTerminology
+    }, {
+        name: "Zip Code"|xl
+        ,required: true
+        ,data_type: dataTypeText
+        ,field_id: "subscriber_postal_code"
+        ,fld_length: 15
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "js-capitalize-me mb-1 js-verify-address"
+        ,skip: useStateTerminology
+    }, {
+        name: "Postal Code"|xl
+        ,required: true
+        ,data_type: dataTypeText
+        ,field_id: "subscriber_postal_code"
+        ,fld_length: 15
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "js-capitalize-me mb-1 js-verify-address"
+        ,skip: not useStateTerminology
+    }, {
+        name: "Country"|xl
+        ,required: true
+        ,data_type: country_data_type
+        ,field_id: "subscriber_country"
+        ,fld_length: 10
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "js-capitalize-me mb-1 sel2"
+        ,list_id: country_list
+    }, {
+        name: "Subsciber Phone"|xl
+        ,required: false
+        ,data_type: dataTypeText
+        ,field_id: "subscriber_phone"
+        ,fld_length: 20
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "mb-1"
+    }, {
+        name: "CoPay"|xl
+        ,required: false
+        ,data_type: dataTypeText
+        ,field_id: "copay"
+        ,fld_length: 6
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "mb-1"
+    }, {
+        name: "Accept Assignment"|xl
```

Comment:
all the `name` entries above should change the xl to xlj (and remove the surrounding quotes)

Reply:
So these are being output via the generate_form_field in php
https://github.com/DiscoverAndChange/openemr/blob/1080317d93050566efce71abf841a62607dd8dd8/templates/patient/insurance/_insurance_edit_screen_edit.html.twig#L103

All the values are escaped in HTML and so we wouldn't want to us xlj here. It may look like this is JS but its actually twig and not used in JS at all.
https://github.com/openemr/openemr/blob/a06dbf9c7314f343e7cde1471ebadacce2894782/library/options.inc.php#L553

Reply:
sounds good. didn't realize this was being set as a variable for twig and not being outputted to js variable.